src/components/collection/Config.tsx

```diff
@@ -45,7 +51,7 @@ function CollectionConfig({
         resourceConfigErrorsExist ||
         fullSourceErrorsExist;
 
-    const hasWarnings = bindingErrorsExist;
+    const hasWarnings = bindingErrorsExist || sourceBackfillRecommended;
```

Comment:
This is now possibly going over way more stuff. I think it is time to move it into a stand alone component.